Makefile

```diff
@@ -2,7 +2,7 @@
 
 
 .npminstall: package.json
-	@if ! [ $(shell command -v npm) ]; then \
+	@if ! [ $(shell which npm) ]; then \
```

Comment:
but when not found it will return something like `bash: type: npms: not found` and then the if statement will always be true, isn't it?

Reply:
At least in Arch Linux works correctly, because the return value (not the string printed) is 1. As far as I know, this must works in every Linux system in the same way (I'm 4/5 about it). I'm 2/5 sure about it must works in MacOS.

Reply:
I just tested and it works fine